File: Python/Fundamentals/04_Functions/Exercise/add_and_subtract.py
num1 = int(input())
num2 = int(input())
num3 = int(input())


def sum_numbers(a, b):
    result = a + b
    return result


def subtract(a, b) -> object:
    result = a - b
    return result


def add_and_subtract(a, b, c):
    # sum = sum_numbers(a, b)
    sub = subtract(sum_numbers(a, b), c)
    return sub


print(add_and_subtract(num1, num2, num3))

File: JS/JS  Fundamentals/04_Functions/exercise/addAndSubtract.js
function addSubtract(num1, num2, num3) {

    function add(n1, n2) {
        return n1 +n2;
    }

    function subtract(n1, n2) {
        return n1 - n2;
    }
    return subtract(add(num1, num2), num3);
}


console.log(addSubtract(23, 6, 10));

console.log(addSubtract(1, 17, 30));

console.log(addSubtract(42, 58, 100));